plugins/lookup/filetree.py

```diff
@@ -42,7 +44,6 @@
     dest: /web/{{ item.path }}
     state: link
     force: yes
-    mode: '{{ item.mode }}'
```

Comment:
Why not add `follow: false` instead? For me, it works with that (i.e. it ignores the mode), while it should also work for filesystem types where link modes can be set.

Reply:
That's way better. I was looking for that but I guess I was too tired to notice that option. It's [right here in the open though](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-follow).